dot-atom/keymap.cson

```diff
@@ -2,7 +2,8 @@
 #
 # Atom keymaps work similarly to style sheets. Just as style sheets use
 # selectors to apply styles to elements, Atom keymaps use selectors to associate
-# keystrokes with events in specific contexts.
+# keystrokes with events in specific contexts. (Unlike style sheets however,
+# each selector can only be declared once.)
```

Comment:
I think I prefer the latter option too – putting the examples inline here would make the comments section really long.